src/classes/GE_PaymentServices.cls

```diff
@@ -56,6 +56,20 @@ public with sharing class GE_PaymentServices {
 
     @TestVisible Map<String, String> config;
 
+    private enum ElevateEndpoints { TOKENIZE, PURCHASE }
+
+    private String getEndpoint(ElevateEndpoints endpoint) {
```

Comment:
We were using this endpoints from a 'test' class that's why they were public. That class needs to be re-visited anyway, just pointing out that we might consider making the enum public at some point in the future.

Reply:
Ah! Thank you for the reminder. I'll revert these to public for now.